ui/src/app/base/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -31,6 +31,7 @@ export type Props<I, PK extends keyof I> = {
   getValueDisplay?: (filter: FilterKey, value: FilterValue) => ReactNode;
   items: I[];
   onUpdateFilterString: (filterString: string) => void;
+  disabled?: boolean;
```

Comment:
We don't actually enforce this with linting, but typically we sort object keys alphabetically so it's easier to find what you're looking for in larger objects (like some of the data returned from the API).

I've added an issue to enforce it here: https://github.com/canonical-web-and-design/maas-ui/issues/3314